examples/md-flexible/src/simulations/MDFlexSingleNode.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file MDFlexSingleNode.h
+ * @author J. Körner
+ * @date 07.04.2021
+ */
+
+#include "MDFlexSingleNode.h"
+
+#if defined(AUTOPAS_INTERNODE_TUNING)
+#include <mpi.h>
+#endif
+
+#include <fstream>
+#include <iostream>
+
+MDFlexSingleNode::MDFlexSingleNode(int argc, char **argv) : MDFlexSimulation(argc, argv) {
+#if defined(AUTOPAS_INTERNODE_TUNING)
+  MPI_Init(&argc, &argv);
+  int rank;
+  MPI_Comm_rank(MPI_COMM_WORLD, &rank);
+  std::cout << "rank: " << rank << std::endl;
+#endif
```

Comment:
My intention was to preserve the original code. But nontheless, you are still able to run the MDFlexSingleNode on multiple processors. MDFlexSingleNode just does do any optimization on top of AutoPas. You were able to do the domain decomposition manually, before (by using different .yaml configurations, I guess). I wanted to preserve this option.
But then one can argue about the names of the classes. I may try to come up with something better.

Reply:
> You were able to do the domain decomposition manually, before (by using different .yaml configurations, I guess)

Not really since there is no communication between the instances of the old version. So it's more a bunch of independent simulations. There is no real value in keeping this "feature" alive ;)